pkg/kapp/crdupgradesafety/change_validator.go

```diff
@@ -71,6 +71,98 @@ func EnumChangeValidation(diff FieldDiff) (bool, error) {
 	return handled(), nil
 }
 
+func MinimumChangeValidation(diff FieldDiff) (bool, error) {
```

Comment:
TODO: Add go doc comments above these functions

Reply:
Done in https://github.com/carvel-dev/kapp/pull/944/commits/4d6469539335a73420c4d88b1bf349a24c6200a6